src/objects.js

```diff
@@ -37,6 +37,8 @@ const createPerson = (name, age) => {
   
     const getAges = (people) => {
         ages = [];
+
+      // no need for declaring the function separately
```

Comment:
You are assigning a function expression `() => { ... }` to a variable in quite a few places. This is handy if you plan to use it elsewhere, but for this example where we're only using it once, it'd be better to make it part of the method call. You can also make use of [Array.prototype.map()](https://gitlab.com/mcrcodes/curriculum/blob/master/bytes/javascript-101/arrays/map.md) here:

```js
return people.map(person => person.age)
```